### PR TITLE
New Querys and Data

### PR DESCRIPTION
**Pull request by miguelesco**
## What's new

- Update Queries answer all the questions required.
- Update the Data to add the Species column.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -38,4 +38,28 @@ WHERE name != 'Gabumon'
...
+/* How many animals are there? */
+SELECT COUNT(*) FROM animals;
+
+/* How many animals have never tried to escape? */
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
+
+/* What is the average weight of animals? */
+SELECT AVG(weight_kg) FROM animals;
+
```
Comment (Davidosky007) [inline queries.sql:120]: * The queries for transactions stated in the project [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction,Commit%20transaction) are missing kindly add them 🙏

File: schema.sql [hunk 2]
```diff
@@ -7,4 +7,9 @@ CREATE TABLE animals (
...
 	neutered BOOLEAN,
 	weight_kg DECIMAL
-);
\ No newline at end of file
+);
+
+/* Add a column species of type string to your animals table. */
+
+ALTER TABLE animals
+ADD COLUMN species text;
```
Comment (Davidosky007) [inline schema.sql]: * Kindly use the right data type of string for the species column [Check requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Add%20a%20column%20species%20of%20type%20string%20to%20your%20animals%20table.%20Modify%20your%20schema.sql%20file.) read more [on data types for strings ](https://www.tutlane.com/tutorial/sql-server/string-data-types-in-sql)

File: data.sql [hunk 3]
```diff
@@ -5,4 +5,10 @@ VALUES
 	(1, 'Agumon', 'Feb 3, 2020', 0, TRUE, 10.23),
 	(2, 'Gabumon', 'Nov 15, 2018', 2, TRUE, 8),
 	(3, 'Pikachu', 'Jan 7, 2021', 1, FALSE, 15.04),
-	(4, 'Devimon', 'May 12, 2017', 5, TRUE, 11)
+	(4, 'Devimon', 'May 12, 2017', 5, TRUE, 11),
+	(5, 'Charmander', 'Feb 8, 2020', 0, FALSE, 11),
+	(6, 'Plantmon', 'Nov 15, 2022', 2, TRUE, 5.7),
+	(7, 'Squirtle', 'Apr 2, 1993', 3, FALSE, 12.13),
+	(8, 'Angemon', 'Jun 12, 2005', 1, TRUE, 45),
```
Comment (Davidosky007) [inline data.sql]: * Kindly correct the highlighted weights as it's a negative one (ie -11, -5.7, -12.13, -45 ) [Check the requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Animal%3A%20His%20name,escape%203%20times.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @miguelesco  👋, 

Good job so far! 💪 
* Nice looking readme file ✔️ 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: queries.sql [hunk 4]
```diff
@@ -38,4 +38,81 @@ WHERE name != 'Gabumon'
...
+/* Inside a transaction update the animals table by setting 
+the species column to unspecified. Verify that change was 
+made. Then roll back the change and verify that species 
+columns went back to the state before tranasction. */
+
+BEGIN;
+	UPDATE animals
+	SET species = 'unspecified';
+ROLLBACK;
```
Comment (Davidosky007) [inline queries.sql]: * I suggest you properly state these insertions `Inside a transaction update the animal's table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.`  
* You only started the transaction, and made the change (` UPDATE animals SET species = 'unspecified';`) but you didn't verify that change was made and undone for example:  ` SELECT species from animals;` 
  ` ROLLBACK;` this verifies change was made.
Then
  ` SELECT species from animals; ` 
`

File: queries.sql [hunk 5]
```diff
@@ -38,4 +38,81 @@ WHERE name != 'Gabumon'
...
+	UPDATE animals
+	SET species = 'digimon'
+	WHERE name LIKE '%mon';
+/* Update the animals table by setting the species column 
+to pokemon for all animals that don't have species already set. */
+
+	UPDATE animals
+	SET species = 'pokemon'
+	WHERE species ISNULL;
+COMMIT;
```
Comment (Davidosky007) [inline queries.sql:74]: * Kindly complete this transaction as the requirement states that you `Verify that change was made` and `persists after commit`. [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Commit%20the%20transaction,persists%20after%20commit.)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @miguelesco 👋, 

Good job so far! 💪 
* Nice looking readme file ✔️ 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi @miguelesco, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**